nipype/utils/nipype2boutiques.py

```diff
@@ -55,53 +57,133 @@ def generate_boutiques_descriptor(
     tool_desc = {}
     tool_desc['name'] = interface_name
     tool_desc[
-        'command-line'] = "nipype_cmd " + module_name + " " + interface_name + " "
+        'command-line'] = interface_name + " "
+    tool_desc['author'] = "Nipype (interface)"
+    if author is not None:
+        tool_desc['author'] = tool_desc['author'] + ", " + author + " (tool)"
     tool_desc[
         'description'] = interface_name + ", as implemented in Nipype (module: " + module_name + ", interface: " + interface_name + ")."
     tool_desc['inputs'] = []
-    tool_desc['outputs'] = []
-    tool_desc['tool-version'] = interface.version
-    tool_desc['schema-version'] = '0.2-snapshot'
-    if docker_image:
-        tool_desc['docker-image'] = docker_image
-    if docker_index:
-        tool_desc['docker-index'] = docker_index
+    tool_desc['output-files'] = []
+    tool_desc['groups'] = []
+    tool_desc['tool-version'] = interface.version if interface.version is not None else "1.0.0"
```

Comment:
Wouldn't it be better to state no version was provided rather than assume this?

Reply:
That's what I thought at first, but then I found it looked weird on Zenodo to not have a version number here:
![image](https://user-images.githubusercontent.com/14361988/55110048-91f7a100-50ad-11e9-8985-ac0f1773e7ac.png)